pandas_transform/pandas_transform.md

```diff
@@ -224,93 +224,81 @@ This data is saved as a [csv file hosted on GitHub](https://raw.githubuserconten
+
+url_contents = open_url(url)
+text = url_contents.read()
+file = io.StringIO(text)
+
+covid_testing = pd.read_csv(file);
+
+"HTML: <a href='" + url + "'>covid_testing</a> has been loaded"
+```
+
+@pyodide_readcsv_explainer
+
+If you're running this code on your computer, you didn't have any output after the last command because we didn't ask it to print anything. But it did create the `covid_testing` DataFrame. We can take a look at with `print(covid_testing)`:
+
+```python
+print(covid_testing)
+
+```
+@Pyodide.eval
 
-Unless you have an extremely wide browser window, the output above is broken into as many columns as it can fit at a time. Some interfaces will output the DataFrame and let you scroll right to see more columns, but this interface just prints as many columns as it can fit and then starts again with the next columns. Good thing it didn't print all 15524 rows!
+Unless you have an extremely wide browser window, the output above will only show as many columns as it can fit at a time. Some interfaces will output the DataFrame and let you scroll right to see more columns, but this interface just prints as many columns as it can fit and then shows you the real dimensions at the bottom. Good thing it didn't print all 15524 rows!
 
 When you print a DataFrame or Series you will see the first five rows and the last five rows of a DataFrame. If, like this one, it has more than 10 rows, the hidden rows will be indicated by ellipses. You can also ask for the first or last 5 rows of data using the methods `.head()` and `.tail()`. Try putting a number in the parentheses to get different numbers of rows.
 
-<div class="python_data">
-<lia-keep>
-<script type="text/x-sage">
-#print(covid_testing.head())
-</script>
-</lia-keep>
-</div>
+```python
+print(covid_testing.head(4))
+
+```
+@Pyodide.eval
 
 
 Loading your own data
 ---
 
-To create a DataFrame of your data, make sure you use the right command for your file type, and make sure the location of the file is in quotes.
-
-| File type | Read command               |
-| --------- | -------------------------- |
-| .csv      | pd.read\_csv('location')   |
-| .xlsx     | pd.read\_excel('location') |
-| .ods      | pd.read\_excel('location') |
-| .json     | pd.read\_json('location')  |
-| .html     | pd.read\_html('location')  |
-| .sql      | pd.read\_sql('location')   |
-
+If you are trying out this code on your own computer, you can make a DataFrame out of any tabular data you have access to! To create a DataFrame of your data, make sure you use the right command for your file type, and make sure the location of the file is in quotes.
 
+| File type | Read command |
+| - | - |
+| .csv | pd.read_csv('location')|
+| .xlsx | pd.read_excel('location')|
+| .ods | pd.read_excel('location')|
+| .json| pd.read_json('location')|
+| .html| pd.read_html('location')|
+| .sql | pd.read_sql('location')|
```

Comment:
Why are we taking out the backslashes before underscores? I know it's not technically necessary, since there's only one underscore per line so it shouldn't italicize anything accidentally, but seems like there's no reason to take the backslashes out, right?

Reply:
I put them back in!